Resources/Prototypes/Catalog/VendingMachines/Inventories/secdrobe.yml

```diff
@@ -4,27 +4,31 @@
     ClothingBackpackSecurity: 3
     ClothingBackpackSatchelSecurity: 3
     ClothingBackpackDuffelSecurity: 3
-    ClothingUniformJumpsuitSec: 3
-    ClothingShoesBootsJack: 3
-    ClothingHeadHatBeretSecurity: 3
+    ClothingHeadHatBeretSecurity: 6 # DeltaV - they always run out... :(
```

Comment:
this needs Delta-V comments for removing the Sec jumpsuit & boots

Reply:
They haven't been removed, just moved down the list to keep things a little prettier. I'll revert this change because it'll probably cause more headaches than it'll save.